Updated nginx configuration for kibana running with context path

SachinKSingh28 opened the pull request:
This change is to be able to fix the issue described in https://github.com/Accenture/adop-nginx/issues/25

Also simplifies the rules that we have in tools-context for kibana.

This is dependent on https://github.com/SachinKSingh28/adop-kibana repository and image produced from it.

anton-kasperovich commented on resources/configuration/sites-enabled/tools-context.conf:
```diff
@@ -95,6 +72,5 @@ server {
         proxy_redirect off;
         proxy_pass http://ldap-phpadmin;
     }
-    include /etc/nginx/sites-enabled/service-extension/*.conf;
```
Why we want to remove this line? It was introduced in https://github.com/Accenture/adop-nginx/pull/22

SachinKSingh28 commented:
@anton-kasperovich : Thanks for the review! I have added the server-extension configuration back in. It was accidentally removed. Also squashed the commits.

SachinKSingh28 commented:
Closing this PR as the same issue has been fixed in PR https://github.com/Accenture/adop-nginx/pull/29

svscorp commented:
Yay!
